react-native-in-app-payments-quickstart/app/screens/HomeScreen.tsx

```diff
@@ -180,24 +194,24 @@ export default function HomeScreen() {
     if (chargeServerHostIsSet()) {
       try {
         await chargeCardNonce(cardDetails.nonce);
-        showAlert(
-          'Your order was successful',
-          'Go to your Square dashbord to see this order reflected in the sales tab.',
+        setValue(
+          'Congratulation, Your order was successful',
+          'Go to your Square dashbord to see this order reflected in the sales tab.', true
         );
       } catch (error: any) {
-        showAlert('Error processing GooglePay payment', error.message);
+        setValue('Error processing GooglePay payment', error.message, false);
       }
     } else {
       printCurlCommand(cardDetails.nonce, SQUARE_APP_ID);
-      showAlert(
+      setValue(
         'Nonce generated but not charged',
-        'Check your console for a CURL command to charge the nonce, or replace CHARGE_SERVER_HOST with your server host.',
+        'Check your console for a CURL command to charge the nonce, or replace CHARGE_SERVER_HOST with your server host.', false
```

Comment:
should this be in a new line? 

We should celebrate the developer made this far. Showing a failure icon isn't encouraging. Can we make it a success message?

Reply:
Done